backend/.flake8

```diff
@@ -6,4 +6,4 @@ per-file-ignores = tests/*:S101
 max-complexity = 10
 max-line-length = 120
 import-order-style = google
-application-import-names = opre_ops,ops_site,tests
+application-import-names = django,ops_site,tests
```

Comment:
Actually, I think the correct thing to do here is keep `opre_ops`, but remove `ops_site` and `django`.  We especially don't want `django` in this list because then any actual import of the `django` framework will be treated as an application-local import and be grouped with our application imports.

Reply:
updated and resolved